config/polarities.json

```diff
@@ -0,0 +1,22 @@
+[{
+  "id": "AP_DEFENSE",
+  "name": "Vazarin"
+}, {
+  "id": "AP_TACTIC",
+  "name": "Naramon"
+}, {
+  "id": "AP_ATTACK",
+  "name": "Madurai"
+}, {
+  "id": "AP_POWER",
+  "name": "Zenurik"
+}, {
+  "id": "AP_PRECEPT",
+  "name": "Penjaga"
+}, {
+  "id": "AP_WARD",
+  "name": "Unairu"
+}, {
+  "id": "AP_UMBRA",
+  "name": "Umbra"
+}]
```

Comment:
add....
```json
{
  "id": "AP_UNIVERSAL",
  "name": "???"
}
```

This matches the text shown on those rivens. I think "None" would also make sense. I know there's a new aura polarity as well, but i don't know what we'd find the `AP_XXXXX` name for it under.